charts/cinder-csi-plugin/values.yaml

```diff
@@ -28,6 +28,10 @@ csi:
       repository: k8s.gcr.io/sig-storage/livenessprobe
       tag: v2.1.0
       pullPolicy: IfNotPresent
+    failureThreshold: 5
+    initialDelaySeconds: 10
+    timeoutSeconds: 3
+    periodSeconds: 10
```

Comment:
I think https://github.com/kubernetes/cloud-provider-openstack/pull/1395/files
is proposing to update those values 
how about make them sync  ? @ramineni @jralbert

Reply:
@jichenjc yes, they should be in sync. 10s perodicity might be overhead as it does API call as well. Appreciate others opinions as well , on what could be optimum default values. @lingxiankong @jralbert ^^

Reply:
It certainly seems right that the defaults should match up between the manifests and the charts. As for optimum default values, I suppose that depends a bit on what common deployment scenarios look like. The OpenStack clouds that I've run have generally been pretty sluggish on API requests, especially those that require referencing multiple resources, so I'd definitely agree with defaults that are a bit more forgiving than the current values. 10 seconds of timeout within a 60 second period seems pretty reasonable; I'll adjust this PR to match @ramineni's.